portality/static/js/application_form.js

```diff
@@ -563,6 +564,11 @@ doaj.af.EditorialApplicationForm = class extends doaj.af.BaseApplicationForm {
         }
         this.form.submit();
     }
+
+    setAllFieldsOptionalIfAppropriate() {
+        return this.statusesNotRequiringValidation.includes(this.jq("#application_status").val());
```

Comment:
I think this means you can now no longer explicitly set "all fields optional" with that checkbox.

It's possible now that we have different behaviours for journal-like and application-like forms that it would be sensible to add an extra class layer between EditorialApplicationForm and the specific implementations that handles validation exceptions according to the form.

Reply:
ok, I'd need to test the equivalent journal form directly, which I can do tomorrow, but it looks like the "make_all_fields_optional" checkbox is no longer looked at anywhere, so the ManEd Journal form won't be able to override validation by selecting that checkbox.

Reply:
I have confirmed this by manual testing.  If you go to the Journal admin form and select the "Allow save without validation" with checkbox, it still validates the form, and does not allow you to save.

Reply:
Cannot reproduce it. I've saved the journal without title:

![image](https://user-images.githubusercontent.com/8298769/104726364-5ff47a80-572b-11eb-9079-8cd9bb6a819f.png)
Confirmed by search:
![image](https://user-images.githubusercontent.com/8298769/104726405-713d8700-572b-11eb-9734-0c53445d95c6.png)
Not saved when checkbox is unchecked:
![image](https://user-images.githubusercontent.com/8298769/104726489-97fbbd80-572b-11eb-9c2a-4e50cc6349a8.png)

Reply:
fixed